ios/Classes/NativeBridge/Handlers/AuthenticatorsHandler.swift

```diff
@@ -115,24 +91,29 @@ extension AuthenticatorsHandler: AuthenticatorRegistrationDelegate {
     }
 }
 
-extension AuthenticatorsHandler: AuthenticatorDeregistrationDelegate {
+class AuthenticatorDeregistrationDelegateImpl: AuthenticatorDeregistrationDelegate {
+    private var deregistrationCompletion: ((Result<Void, FlutterError>) -> Void)
```

Comment:
I think the name `completion` is enough - you don't have another block within this class, do you? And why is it `var`? If this is set only in `init` and you are not going to change it should be `let`